pipelines/katello_pipeline.yml

```diff
@@ -25,10 +25,12 @@
     - foreman_repositories
     - katello_repositories
     - role: foreman_client_repositories
-      when: katello_version == "nightly" or katello_version is version('3.9', '>=')
+      when: katello_version == "nightly" or katello_version is number and katello_version is version('3.9', '>=')
     - disable_firewall
     - haveged
     - update_os_packages
+    - role: foreman_installer_devel_scenario
+      when: katello_version == "devel"
     - foreman_installer
```

Comment:
does this run both the devel installer and the normal installer?

Reply:
It does both, just like the [devel playbook](https://github.com/theforeman/forklift/blob/master/playbooks/devel.yml#L21)

Reply:
The [foreman_installer_devel_scenario](https://github.com/theforeman/forklift/blob/master/roles/foreman_installer_devel_scenario/tasks/main.yml) role doesn't actually run the installer.

Reply:
ah ok, didn't realize we do both